server/src/main/java/com/linecorp/centraldogma/server/internal/api/TokenService.java

```diff
@@ -192,6 +192,31 @@ public CompletableFuture<Token> updateToken(ServiceRequestContext ctx,
         );
     }
 
+    /**
+     * PATCH /tokens/{appId}/permission
+     *
+     * <p>Updates a permission of a token of the specified ID.
+     */
+    @Patch("/tokens/{appId}/level")
+    @Consumes("application/json-patch+json")
+    @RequiresAdministrator
+    public CompletableFuture<Token> updateTokenLevel(ServiceRequestContext ctx,
+                                                     @Param String appId,
+                                                     Author author, User loginUser) {
+        return getTokenOrRespondForbidden(ctx, appId, loginUser).thenCompose(
+                token -> {
+                    if (token.isAdmin()) {
```

Comment:
Should we make this API idempotent instead of changing the behavior depending on the token's status?
Some people may want to retry an update request if the previous request is timed out somehow.

How about taking the desired level in the body?
```
PATCH /tokens/{appId}/permission 

{ "level": "USER" }
```

Reply:
@ikhoon 
Oh, i got it. 
In this project, in which package should the request body class be placed?

```
public class TokenLevel {
    private String level;

    public TokenLevel() {
    }

    public String getLevel() {
        return level;
    }

    public void setLevel(String level) {
        this.level = level;
    }
}
```

Reply:
Okay, Thanks for your comment. I'll add it.